cmd/clusterctl/cmd/plugin_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"github.com/Masterminds/goutils"
+	. "github.com/onsi/gomega"
+)
+
+var baseArgs = []string{"clusterctl", "foo"}
+
+const (
+	forkEnvVar          = "FORK"
+	pluginCommandEnvVar = "PLUGIN_COMMAND"
+)
+
+func Test_plugin(t *testing.T) {
+	if goutils.DefaultString(os.Getenv(forkEnvVar), "false") == strconv.FormatBool(false) {
```

Comment:
```suggestion
	if os.Getenv(forkEnvVar) == "false" {
```

This value is only used in this test, so I think it's enough to compare as a string. It reduces the dependency.

Reply:
This is not equivalent for the initial (FORK env var is empty) test configuration.
I will convert to the following code, to reduce dependencies:
```golang
if os.Getenv(forkEnvVar) == "" {
```

Reply:
sorry you're right 👍

Reply:
Fixed, please review.

Reply:
I'd like to understand this some more, did this result in an issue in our CI? Because locally it seems fine for me.